.thoth.yaml

```diff
@@ -7,7 +7,7 @@ runtime_environments:
     operating_system:
       name: fedora
```

Comment:
please update these to ubi as well.

Reply:
thanks

Reply:
https://github.com/thoth-station/qeb-hwt/pull/133/commits/273ff38709c8f81ac48f0b11222622345e9ece36